drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -274,23 +311,30 @@ function DragNDropTemplates(url_name) {
                         ),
                         renderCollection(zoneTemplate, ctx.zones, ctx),
                         renderCollection(itemTemplate, items_placed_unaligned, ctx)
-                    ]
-                    ),
+                    ]),
                 ]),
-                keyboardHelpTemplate(ctx),
+                h("section.actions-toolbar", {}, [
+                    sidebarTemplate(ctx), // sidebar has float:right, so putting it before the content to float
+                    (ctx.show_submit_answer ? submitAnswerTemplate(ctx) : null)
+                ]),
+                h("div", {style: {clear: 'both'}}), // div to clear floating
```

Comment:
Can we get rid of this redundant element? Perhaps instead of floating the sidebar position it absolutely with `right: 0`.